website/azurerm.erb

```diff
@@ -174,6 +174,15 @@
               </ul>
             </li>
 
+            <li<%= sidebar_current("docs-azurerm-resource-image") %>>
+              <a href="#">Image Resources</a>
```

Comment:
I put it under "Managed Disk" to match the sidebar [here](https://docs.microsoft.com/en-us/rest/api/manageddisks/), hope that's Ok; agree that we should do a bit of re-organization of the side bar